virttest/qemu_devices/utils.py

```diff
@@ -73,3 +73,18 @@ def none_or_int(value):
         return int(value)
     else:
         raise TypeError("This parameter has to be int or none")
+
+
+def mark_qemu_command_style(config, key, dev):
+    """
+    mark the qemu command style based on the settings
+    :param config - dict with setting
+    :param key - the field of device
+    :param dev - the device needed to be marked
+    :return dev - device with qemu command style
+    """
+    if key in config:
+        name = dev.cmdline().split(" ")[0].split("-")[1]
```

Comment:
Instead of getting the cmdline category by rendering the raw format cmdline every time, which I think is not that efficient, I'd suggest using other approaches, for example, via getting the class type.